app/src/wavevr/cpp/DeviceDelegateWaveVR.cpp

```diff
@@ -657,8 +657,17 @@ DeviceDelegateWaveVR::SetReorientTransform(const vrb::Matrix& aMatrix) {
 }
 
 void
-DeviceDelegateWaveVR::Reorient(vrb::Matrix& transform) {
-  m.reorientMatrix = DeviceUtils::CalculateReorientationMatrixWithoutRoll(transform, GetHeadTransform().GetTranslation());
+DeviceDelegateWaveVR::Reorient(vrb::Matrix& transform, ReorientMode mode) {
```

Comment:
Aren't these Reorient functions identical in all the implementations of the DeviceDelegate interface ? If that's the case, we may want to implement in the interface declaration. I know it's perhaps a too complex code to be in a header file, but it seems quite annoying to have such code duplicated in 3 (for now) different classes.

Reply:
I agree, but the code was the same before as well. I'd suggest to fix that in a separate PR, because it might involve additional refactorings.